xml/vm_security.xml

```diff
@@ -0,0 +1,226 @@
+</para>
+</note>
+</abstract>
+<dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+<dm:bugtracker/>
+<dm:translation>yes</dm:translation>
+</dm:docmanager>
+</info>
+<sect1 xml:id="vm-security-hardware-support">
+<title>Supported hardware</title>
+
+<para>
+A system with an AMD EPYC (3rd Gen or newer) is required to run AMD SEV-SNP virtual machines. The BIOS of the AMD machine must provide the necessary options to enable support for confidential computing on the platform.</para>
+</sect1>
+<sect1 xml:id="vm-security-enable-confidential-compute-module">
+<title>Enabling confidential compute module</title>
+
+<para>The necessary packages for AMD SEV-SNP feature are shipped via a confidential compute module. You must enable it at system installation time or later via the SUSEConnect command-line tool.</para>
+<itemizedlist>
+<listitem>
+<para>To check whether the module is already enabled, run the command:
+</para>
+<screen> &prompt.sudo; suseconnect -l</screen>
+<para>This displays the list of available modules with their activation status and commands to enable the inactive modules.</para>
+<para>The inactive confidential compute module appears as given below:</para>
+<screen>Confidential Computing Technical Preview Module 15 SP6 x86_64
+Activate with: suseconnect -p sle-module-confidential-computing/15.6/x86_64</screen>
+</listitem>
+<listitem> <para>To enable the confidential computing module technology preview, run the command:</para>
+<screen> &prompt.sudo; <command>suseconnect -p sle-module-confidential-computing/15.6/x86_64</command>
+Registering system to SUSE Customer Center
+Updating system details on https://scc.suse.com ...
+Activating sle-module-confidential-computing 15.6 x86_64 ...
+Adding service to system ...
+Installing release package ...
+Successfully registered system</screen>
+<para>The confidential compute module is enabled and you can install the packages.</para>
+</listitem>
+</itemizedlist>
+</sect1>
+<sect1 xml:id="vm-security-verify-setup">
+<title>Installing packages and setting up the base system</title>
+
+<para>
+The confidential compute module provides replacement packages supporting AMD SEV-SNP. To ensure a maximum of compatibility, these packages are based on the code streams from &productname; &productnumber;.</para>
+<para>The three components that need to be replaced are:</para>
+<itemizedlist>
+<listitem>
+<para>The Linux kernel</para>
+</listitem>
+<listitem>
+<para>QEMU Virtual Machine Monitor</para>
+</listitem>
+<listitem>
+<para>&libvirt; framework</para>
+</listitem>
+</itemizedlist>
+<procedure>
+<step>
+<para>To install the replacement packages, run the command:</para>
+<screen>
+&prompt.sudo; <command>zypper install --from SLE-Module-Confidential-Computing-15-SP6-Pool --from SLE-Module-Confidential-Computing-15-SP6-Updates qemu libvirt</command></screen>
```

Comment:
please add at the end of the commande line: kernel-coco

Reply:
Added:

<command>zypper install --from SLE-Module-Confidential-Computing-15-SP6-Pool --from SLE-Module-Confidential-Computing-15-SP6-Updates qemu libvirt kernel-coco</command>